### PR TITLE
Prezi case study

### PR DESCRIPTION
**Pull request by VioletteLepercq**
@jeffboudier the blog post has been enhanced thanks to Moritz and Maté's validation. 
Can you please publish the youtube video when u r back so we can release the blog post?

### CODE REVIEW

File: prezi-case-study.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- user: MoritzLaurer
+- user: bmateusz
+  guest: true
+---
+
+# Going multimodal: How Prezi is leveraging the Hub and the Expert Support Program to accelerate their ML roadmap
+
+Everybody knows that a great visual is worth a thousand words. The team at Prezi, a visual communications software company, is putting this insight into practice with their Prezi presentations that combine images and text in highly dynamic presentations. 
+
+Prezi has joined the Hugging Face Expert Support Program to leverage modern machine learning's potential fully. Over the past months, Hugging Face has supported Prezi in integrating smaller, more efficient open-source models into their ML workflows. This cooperation started at a perfect time, as multimodal models are becoming increasingly capable.
```
Comment (MoritzLaurer) [inline prezi-case-study.md:16]: "to fully leverage modern ..."

**Review by MoritzLaurer — APPROVED** [review]
Great, LGTM!

File: prezi-case-study.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Prezi has joined the Hugging Face Expert Support Program to fully leverage modern machine learning's potential. Over the past months, Hugging Face has supported Prezi in integrating smaller, more efficient open-source models into their ML workflows. This cooperation started at a perfect time, as multimodal models are becoming increasingly capable. 
+
+We recently sat down with [Máté Börcsök](https://www.linkedin.com/in/mateborcsok/?originalSubdomain=hu), a backend engineer at [Prezi](https://prezi.com/), to talk about their experience in the [Expert Support Program](https://huggingface.co/support). In this short video, Máté walks us through some of their machine learning work and shares their experience collaborating with our team via the Expert Support Program. 
+
+<iframe width="100%" style="aspect-ratio: 16 / 9;"src="https://www.youtube.com/watch?v=pM6D0tRoIbI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (VioletteLepercq) [inline prezi-case-study.md:20]: @MoritzLaurer or @pcuenca can you confirm that the iframe is correctly written? I do not want to merge before I am sure the video will be displayed correctly on the blog article.

Comment (MoritzLaurer) [reply prezi-case-study.md:20]: I think the youtube link needs to be changed to "https://www.youtube.com/embed/pM6D0tRoIbI"

<iframe width="100%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/pM6D0tRoIbI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

Comment (VioletteLepercq) [reply prezi-case-study.md]: I committed the changes :)

Comment (VioletteLepercq) [reply prezi-case-study.md]: waiting for @pcuenca ultimate approval just to make sure that the video is correctly framed now :)

Comment (pcuenca) [reply prezi-case-study.md]: The video iframe looks good to me!

Comment (VioletteLepercq) [reply prezi-case-study.md]: thanks :)

**Pull request merged**